suave/offchain/offchain.go

```diff
@@ -0,0 +1,26 @@
+package offchain
+
+import (
+	"github.com/ipfs/kubo/client/rpc"
+	iface "github.com/ipfs/kubo/core/coreiface"
+)
+
+type Env struct {
```

Comment:
`Env` feels a little non-descriptive, as in it doesn't really convey a meaning (offchain.Env) 🤔 

What is this for, eventually?

Reply:
It's meant to contain the "offchain environment", i.e. the collection of APIs and services that we export to SUAPPs for off-chain data/computation.  Ordinarily, I'd agree that `Env` doesn't convey a lot of meaning, but I think it actually is an "environment" in this case.